### PR TITLE
 IVP 39: Update About Route

### PR DESCRIPTION
**Pull request by simpsbri**
Updated About route to include JSON requirement with additional information requested. 
Per story request:

JSON data {name, age, asl}
status 200 response

### CODE REVIEW

**Comment by dandigangi** [comment]
Hey can you add the specific requirements into the description?

**Comment by simpsbri** [comment]
Per story request:

- JSON data {name, age, asl}
- status 200 response

File: index.js [hunk 1]
```diff
@@ -109,7 +109,11 @@ app.get('/', (req, res) => {
 app.get('/about', (req, res) => {
   // Sends HTTP response
   // https://expressjs.com/en/5x/api.html#res.send
-  res.send('<h2>res.send()</h2>')
+  res.status(200).json({
+    name: 'Dan',
+    age: '29',
```
Comment (dandigangi) [inline index.js]: Should this be a number not string?

**Review by dandigangi — CHANGES_REQUESTED** [review]
Overall looks good. Left a comment though.

File: index.js (hunk 1 above)
Comment (simpsbri) [reply index.js]: It should be. Updates will be pushed shortly.

**Review by dandigangi — APPROVED** [review]
LGTM

**Pull request merged**